way_to_home/static/src/components/userSettingsForm/notifications/newNotificationItem/newNotificationItem.js

```diff
@@ -0,0 +1,174 @@
+import React, {Component} from 'react';
+
+import TextField from '@material-ui/core/TextField';
+import DeleteIcon from '@material-ui/icons/Delete';
+import Tooltip from '@material-ui/core/Tooltip';
+import Done from '@material-ui/icons/Done';
+import IconButton from '@material-ui/core/IconButton';
+import MenuItem from "@material-ui/core/MenuItem/MenuItem";
+
+import './newNotificationItem.css';
+import axios from "axios";
+
+class NewNotificationItem extends Component{
+
+    state = {
+        time: "08:30",
+        week_day: 0,
+        isTimePickerOpen: false,
+        day: 'Понеділок',
+        days: [
+            {
+                value: 0,
+                label: 'Понеділок',
+            },
+            {
+                value: 1,
+                label: 'Вівторок',
+            },
+            {
+                value: 2,
+                label: 'Середа',
+            },
+            {
+                value: 3,
+                label: 'Четвер',
+            },
+            {
+                value: 4,
+                label: 'Пятниця',
+            },
+            {
+                value: 5,
+                label: 'Субота',
+            },
+            {
+                value: 5,
+                label: 'Неділя',
+            },
+        ],
+        clock: []
+    };
+
+    formatDate = (date) => {
+        let d = date,
+            month = '' + (d.getMonth() + 1),
+            day = '' + d.getDate(),
+            year = d.getFullYear();
+
+        if (month.length < 2) month = '0' + month;
+        if (day.length < 2) day = '0' + day;
+
+        return [year, month, day].join('-');
+    };
+
+    addTime = () => {
```

Comment:
WTF

Reply:
this function generates hours and time for input time selector when we create new notification